schemas/EGA.experiment.json

```diff
@@ -2,12 +2,324 @@
+            "examples": [ "MC-4C", "UMI-4C", "single nucleus RNA sequencing", "RNA-seq of coding RNA from single cells", "Quant-seq", "DNA-seq", "ATAC-seq", "DNase Hi-C", "scDNase-seq", "in situ HiC", "exome sequencing" ]
+          }
+        },
+        "anyOf": [
+          {
+            "title": "Assay subtypes match DNA/RNA assays",
+            "properties": {
+              "assay_subtype": {
+                "anyOf": [
+                  {
+                    "title": "DNA-Assay subtype controlled vocabulary (CV) list",
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_dna.json"
+                  },
+                  {
+                    "title": "RNA-Assay subtype controlled vocabulary (CV) list",
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_rna.json"
+                  }
+                ]                 
+              }
+            }
+          },
+          {
+            "title": "Assay subtypes match array/sequencer assays",
+            "oneOf": [
+              {
+                "title": "Assay type and subtype terms are from the array CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_array.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_array.json" 
+                  }
+                }
+              },
+              {
+                "title": "Assay type and subtype terms are from the sequencer CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_sequencer.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_sequencer.json" 
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+
+      "assayed_molecule": {
+        "type": "string",
+        "title": "Assayed molecule of the experiment [EFO:0002772]",
+        "description": "The human readable ID (e.g. DNA assay), chosen from a list of controlled vocabulary (CV), of the assayed molecule. If you cannot find your term in the CV list, please create an issue at our [metadata GitHub repository](https://github.com/EbiEga/ega-metadata-schema) proposing its addition.",
```

Comment:
```suggestion
        "description": "The biomolecule type assayed in the experiment. If you cannot find your term in the CV list, please create an issue at our [metadata GitHub repository](https://github.com/EbiEga/ega-metadata-schema) proposing its addition.",
        "examples": ["genomic DNA", "messenger RNA"],
```

Reply:
"human readable" is redundant - everything should be human-readable :P

Reply:
True, I used that phrase some times when trying to refer to the human's term for a concept or identifier, something that they can understand. It is indeed redundant in this case 👍